bot/modules/nostr/events.js

```diff
@@ -0,0 +1,35 @@
+const Nostr = require('nostr-tools');
+
+const KIND = {
+  ORDER_CREATED: 20100,
+};
+
+exports.orderCreated = order => {
+  const sk = process.env.NOSTR_SK;
+  const pubkey = process.env.NOSTR_PK;
```

Comment:
I think is not necessary to save the pubkey as an env var but derivate it from the secret key as you did in  `configure()` is the best approach, please move that derivation to here